src/components/Api.js

```diff
@@ -1,3 +1,145 @@
 export default class Api {
-  
-}
\ No newline at end of file
+  constructor() {
+    this.baseURL = "https://around.nomoreparties.co";
+  }
+
+  getUserInfo () {
+    return fetch(`${this.baseURL}/v1/group-12/users/me`, {
+      headers: {
+        authorization: "397bd50b-9f39-4bee-ad9c-11e69aa20ec4"
+      }
+    })
+    .then(response => {
+      if (!response.ok) {
+        return Promise.reject(`Error: ${response.status}`);
+      }
+
+      return response.json();
+    })
+    .catch(err => {
+      console.error(err);
+    })
+  }
+
+  getInitialCards() {
+    return fetch(`${this.baseURL}/v1/group-12/cards`, {
```

Comment:
`/v1/group-12` should be inside `baseURL` as well not to duplicate it in every request